explore_node:
  ros__parameters:
    robot_base_frame: base_footprint
    return_to_init: true
    costmap_topic: global_costmap/costmap
    costmap_updates_topic: global_costmap/costmap_updates
    visualize: true # false
    planner_frequency: 0.2
    progress_timeout: 15.0
    potential_scale: 1.0 #5.0
    orientation_scale: 0.0
    gain_scale: 3.0 #1.0
    transform_tolerance: 0.1
    min_frontier_size: 0.1 #0.6

    map_file: /home/ruro/nav2_ws/src/reto-ciberfisicos/maps/mapa.yaml